supabase/migrations/20260711094204_align_replica_source_schema.sql

```
@@ -0,0 +1,66 @@
+-- Keep local migrations aligned with the primary schema used by the read replica.
```

SQLFluff (4.2.2)

[error] Line is too long (82 > 80).

(LT05)